ragaai_catalyst/tracers/exporters/ragaai_trace_exporter.py

```
@@ -24,6 +23,29 @@
     logger.setLevel(logging.DEBUG) if os.getenv("DEBUG") == "1" else logging.INFO
 )
 
+class TracerJSONEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, datetime):
```

Ruff (0.11.9)

Undefined name `datetime`

(F821)